sql/driver/driver_it_test.go

```diff
@@ -527,6 +528,48 @@ func TestContextCancelAfterFirstPage(t *testing.T) {
 	})
 }
 
+func TestClusterShutdownDuringQuery(t *testing.T) {
+	it.SkipIf(t, "hz < 5.0")
+	ctx := context.Background()
+	port := it.NextPort()
+	cls := it.StartNewClusterWithConfig(it.MemberCount(), it.SQLXMLConfig(t.Name(), "localhost", port), port)
+	defer cls.Shutdown()
+	config := cls.DefaultConfig()
```

Comment:
I think this should be `cls.DefaultConfigWithNoSSL()`. Could you try with `ENABLE_SSL=1`?

Reply:
You are right, thanks https://github.com/hazelcast/hazelcast-go-client/pull/862/commits/680f341c9f032567ffe5fa090f962b9ee2ec064b